### PR TITLE
{bio}[SYSTEM] GSEA v4.0.3

### PR DESCRIPTION
**Pull request by rtripath89**

### CODE REVIEW

**Comment by rtripath89** [comment]
@boegel : Hi, can I get a little clarification on these errors, as it looks like it's code style errors, but I can't figure our, where I need to make modifications for it.

File: easybuild/easyconfigs/g/GSEA/GSEA-4.0.3.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+
+# By default this software uses 4G RAM, so you would need to modify gsea.sh, gsea-cli.sh and gsea_hidpi.sh to use more or less memory.
```
Comment (lexming) [inline easybuild/easyconfigs/g/GSEA/GSEA-4.0.3.eb:6]:
```suggestion
# By default this software uses 4G RAM, so you would need to modify gsea.sh,
# gsea-cli.sh and gsea_hidpi.sh to use more or less memory.
```

**Review by lexming — DISMISSED** [review]
The style checks fail because the line I highlighted here below is too long. On the other hand, will GSEA work in the `SYSTEM` toolchain with no dependencies at all? I'm not familiar with it.

**Comment by rtripath89** [comment]
> The style checks fail because the line I highlighted here below is too long. On the other hand, will GSEA work in the `SYSTEM` toolchain with no dependencies at all? I'm not familiar with it.

It should ...... it does depend on Java, so I have added it as a dependency, in case a system doesn't contain Java.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3501.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/e3f78059962863bd152628a5fd88c7f4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2710.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/ef836cd6885b89c5a6ac2fabf9a59dda for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @rtripath89!

**Pull request merged**